include/zephyr/posix/stropts.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 Abhinav Srivastava
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_POSIX_STROPTS_H_
+#define ZEPHYR_INCLUDE_POSIX_STROPTS_H_
+
+#define RS_HIPRI BIT(0)
+
+typedef struct {
+    int maxlen;
+    int len;
+    char *buf;
+} strbuf;
+
+int putmsg(int fileDescriptor, const strbuf *ctlPtr, const strbuf *dataPtr, int flags);
```

Comment:
I think the convention for function parameters and variable names is not to use camel case.

The names used in the spec would be fine too.
```suggestion
int putmsg(int fildes, const strbuf *ctlptr, const strbuf *dataptr, int flags);
```

Reply:
unresolving this as it isn't resolved